rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -157,12 +157,6 @@ impl std::fmt::Debug for LodeSphericalExpansion {
 
 impl LodeSphericalExpansion {
     pub fn new(parameters: LodeSphericalExpansionParameters) -> Result<LodeSphericalExpansion, Error> {
-        if parameters.potential_exponent > 6 {
-            return Err(Error::InvalidParameter(
-                "LODE is only implemented for potential_exponent <= 6".into()
-            ));
-        }
-
```

Comment:
I would leave at least one validation check, otherwise the user will get an "internal error" from the panic line 298

Reply:
We can leave this check yes.